vendor/github.com/spf13/cobra/projects_using_cobra.md

```diff
@@ -25,7 +25,7 @@
 - [Istio](https://istio.io)
 - [Kool](https://github.com/kool-dev/kool)
 - [Kubernetes](https://kubernetes.io/)
-- [Kubescape](https://github.com/armosec/kubescape)
```

Comment:
To change this, you'd have to do it here: https://github.com/spf13/cobra/blob/main/projects_using_cobra.md?plain=1#L29, and bump the spf13/cobra dependency when a new release thereof is out.

Reply:
Sorry, I did not notice this was in the vendor dir, thank you for pointing it out.